src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -268,7 +291,7 @@ object ScanProcessor extends CommandProcessor {
     }
     var externalConfigAndRules = getEmptyConfigAndRule
     if (config.externalConfigPath.nonEmpty) {
-      externalConfigAndRules = parseRules(config.externalConfigPath.head, lang)
+      externalConfigAndRules = parseRules(config.externalConfigPath.head, lang, true)
```

Comment:
for default values a good practise is to assign it via variable name. Ex= `isExternal = true`